common/scala/src/main/resources/logging.conf

```diff
@@ -1,4 +1,4 @@
 akka {
-  loglevel = "DEBUG"
+  loglevel = "INFO"
```

Comment:
never-mind - thanks for pointing out this is lowering log level not increasing it.